Graphics engine basic prototype

Lduperron opened the pull request:
Adds a various assets (some of which have a TODO to change to better ones)
Sets up various screens.  We still need better buttons and stuff, but I feel it's workable.  Kinda sorta maybe, I hope.
Mocks up the two scenes, still need to be done better with better assets too
Needs to be refactored some to provide callbacks for changing / dynamic functionality, but that should be fairly easy.
Updates the version of libGDX used

Commit:
adds my ideas for ServerEngine

Commit:
Merge branch 'skeleton' into skeletonServer

Conflicts:
	src/main/java/csci331/team/red/core/ServerEngine.java

Commit:
basic implementation of the ServerEngine. Untested.

Commit:
Merge branch 'master' into skeletonServer

Lduperron committed:
Updates the version of libGDX used
Adds a various assets (some of which have a TODO to change to better ones)
Sets up various screens.  We still need better buttons and stuff, but I feel it's workable.  Kinda sorta maybe, I hope.
Mocks up the two scenes, still need to be done better with better assets too
Needs to be refactored some to provide callbacks for changing / dynamic functionality, but that should be fairly easy.

Lduperron committed:
Updates the version of libGDX used
Adds a various assets (some of which have a TODO to change to better ones)
Sets up various screens.  We still need better buttons and stuff, but I feel it's workable.  Kinda sorta maybe, I hope.
Mocks up the two scenes, still need to be done better with better assets too
Needs to be refactored some to provide callbacks for changing / dynamic functionality, but that should be fairly easy.

Commit:
weekend commit

FencaR commented:
What buttons do you need? Or I should say, what images for buttons would you like?

Lduperron committed:
Merge branch 'master' of https://github.com/ojourmel/csci331-team-red into clientEngine

Lduperron committed:
Merge branch 'clientEngine' of https://github.com/ojourmel/csci331-team-red into clientEngine

Lduperron committed:
Merge branch 'skeletonServer' of https://github.com/ojourmel/csci331-team-red into clientEngine

Lduperron committed:
Puts the code in a semi-workable state for a demo, however, some shortcuts taken will require major refactoring.  Later.
Java Enums kinda suck compared to c# ones.
Adds dialog that talks about each game for a little bit, and.. uh, I think we can make stretch 5 minutes if we talk about it as we go.  Maybe.  Someone else want to look at it?

Lduperron commented:
Er, I'm sorry I didn't respond sooner.  Proper themed graphics could work,
but we don't need them in an immediate sense (and don't really know what to
get for them either).  Thanks for doing what you have so far!

On Sun, Oct 20, 2013 at 11:37 PM, FencaR notifications@github.com wrote:

> What buttons do you need? Or I should say, what images for buttons would
> you like?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ojourmel/csci331-team-red/pull/13#issuecomment-26695399
> .

Lduperron committed:
Makes the enums work.  <?> yay (and a bit of invoke)
Removes unused imports

Lduperron committed:
Makes the phone wrap, adds a bit of javadocs.

ojourmel commented:
Waiting on Callback code, and Network/Server integration.
